### ISSUE TITLE
DPS5005 black screen when firmware is compiled with gcc 7.3.0

### ISSUE THREAD

**Issue by bhuism** [kanflo/opendps#132]
My DPS5005 will not start, so it seems, just a black screen, when the firmware is compiled with gcc 7.3.0. It does work with the awesome docker build script.

**Comment by Xenoamor** [comment]
Hmm.. that's interesting
Perhaps this a good chance to get Travis running and have it add releases to the github page

**Comment by kanflo** [comment]
Yes, we really need CI. I have been thinking of setting up a CI in my workshop with DPS hardware, STLink, every single version of gcc, ... and test building, flashing, upgrading and operations.

**Comment by nmaas87** [comment]
@kanflo Drone.io could be a good choice here, they do free plans for OpenSource projects and integrate with Github https://cloud.drone.io/ :)
